packages/terra-application-name/tests/wdio/application-menu-name-spec.js

```diff
@@ -1,16 +1,12 @@
 Terra.describeViewports('ApplicationMenuName', ['tiny', 'medium'], () => {
-  describe('Displays a default application menu name', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-application-name/application-name/application-menu-name-default');
-      browser.pause(3000); // to display image
-    });
-
-    Terra.it.validatesElement({ selector: '#default' });
+  it('Displays a default application menu name', () => {
+    browser.url('/raw/tests/terra-application-name/application-name/application-menu-name-default');
+    browser.pause(3000); // to display image
+    Terra.validates.element({ selector: '#default' });
```

Comment:
```suggestion
    Terra.validates.element('default', { selector: '#default' });
```
wdio 6 will require a screenshot name to be provided